hack/dind-cluster.sh

```diff
@@ -666,8 +689,7 @@ NODE_PREFIX="${CLUSTER_ID}-node-"
 NODE_COUNT=2
 NODE_NAMES=()
 
-BRIDGE_PREFIX="docker"
-BRIDGE_START_NUM=10
+ADDITIONAL_BRIDGE_NAME="${ADDITIONAL_BRIDGE_NAME:-docker10}"
```

Comment:
maybe shouldn't use "docker" in the name since it's not created/managed by docker